pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/prerequisites/index.md

```diff
@@ -19,7 +19,31 @@ Before you begin using Konvoy, you must have:
 
 <p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</p>
 
-If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
+## Control plane nodes
+
+You should have at least three control plane nodes.
+Each control plane node should have at least:
+
+- 4 cores
+- 16 GiB memory
+- Approximately 80 GiB of free space for the volume used for /var/lib/kubelet and /var/lib/containerd.
+- Disk usage must be below 85% on the root volume.
+
+DKP on AWS defaults to deploying an `m5.xlarge` instance with an 80GiB root volume for control plane nodes, which meets the above requirements.
+
+## Worker nodes
+
+You should have at least four worker nodes. The specific number of worker nodes required for your environment can vary depending on the cluster workload and size of the nodes.
+Each worker node should have at least:
+
+- 8 cores
+- 32 GiB memory
```

Comment:
All this sound like nice defaults. However in general a single node should not really matter in a K8s environment but the amount of resources needed by the Pods we're deploying by default...

So given this what we want from the user is:
- minimal 32 cores
- 128GB memory

When it comes to redundancy in AWS we should state that we expect these minimal resources being evenly distributed in at least 3 AZs. 

@dkoshkin I'm having a hard time getting used to this 4 nodes by default thing reason is that if we utilize more than three workers we'll have a redundancy problem anyways.